x-pack/legacy/plugins/ml/server/models/bucket_span_estimator/bucket_span_estimator.d.ts

```diff
@@ -0,0 +1,14 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { IScopedClusterClient } from 'src/core/server';
+import { BucketSpanEstimatorData } from '../../../public/application/services/ml_api_service';
+
+export function estimateBucketSpanFactory(
+  callAsCurrentUser: IScopedClusterClient['callAsCurrentUser'],
+  callAsInternalUser: IScopedClusterClient['callAsInternalUser'],
```

Comment:
NIT: Should use `APICaller` interface instead (same in other `.d.ts` file)
```ts
import { APICaller } from 'src/core/server';

export function estimateBucketSpanFactory(
  callAsCurrentUser: APICaller,
  callAsInternalUser: APICaller,
```

Reply:
Updated in f0bac33e423d5a570bfa2ba8157a9d1e677c6afb